documentation/opencast-installation.md

```diff
@@ -133,6 +133,47 @@ Additionally you must set the label for the annotation publication channel. Add
 
     annotation=Annotation Tool
 
+### Security Configuration
+
+In order to be able to access the tool,
+you need to make some additions to your Spring Security configuration.
+In a single-tenant setup,
+this can be found at `<opencast-etc>/security/mh_default_org.xml`.
+If you have a multi-tenant setup,
+you probably know where this configuration needs to go
+better than I do. :wink:
+
+What you need is something to the effect of the following two lines:
+
+    <sec:intercept-url pattern="/annotation-tool/**" access="ROLE_ANONYMOUS" />
+    <sec:intercept-url pattern="/extended-annotations/**" access="ROLE_ANONYMOUS" />
```

Comment:
We still have this rule:

```xml
<sec:intercept-url pattern="/extended-annotations/reset" access="ROLE_ADMIN" />
```

I have not followed the development closely. Is the `reset` endpoint still in existence?

Reply:
Nope, it's gone: #210